common/xdc/event_generator.go

```diff
@@ -239,7 +304,7 @@ func (g *EventGenerator) getNextPossibleVertex() Vertex {
 		}
 		neighbors := g.connections[leaf]
 		for _, nextV := range neighbors {
-			if nextV.GetCondition() == nil || (nextV.GetCondition() != nil && nextV.GetCondition()(g.previousVertexes)) {
+			if nextV.GetCondition() == nil || (nextV.GetCondition() != nil && nextV.GetCondition()()) {
 				isAccessible = true
 				break
```

Comment:
once `isAccessible == true` this function can basically return?

Reply:
Yes. This function is to check if there is a path to move forward. I updated the break to a reurn